bedrock/firefox/templatetags/helpers.py

```diff
@@ -164,7 +165,8 @@ def download_firefox(ctx, channel='release', platform='all',
         'show_ios': show_ios,
         'alt_copy': alt_copy,
         'button_color': button_color,
-        'download_location': download_location
+        'download_location': download_location,
+        'fluent_l10n': fluent_l10n([locale, 'en'], settings.FLUENT_DEFAULT_FILES)
```

Comment:
you should be able to grab the `fluent_l10n` out of the `ctx`.

```suggestion
        'fluent_l10n': ctx['fluent_l10n'],
```

Reply:
Nice, good suggestion!

Reply:
I'm currently struggling with the download button tests since making this suggested change. Any pointers for passing this as a context variable to the tests that use `RequestFactory()`?

Reply:
Ok, took a step back and now I think I have this figured out

Reply:
Okay. Cool. Please ping me if you want to talk it through.

Reply:
If you have some time, please feel free to take a look at the follow up commit I just pushed?